engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/EnsureCompiledJob.scala

```diff
@@ -112,6 +112,8 @@ final class EnsureCompiledJob(
         applyEdits(new File(module.getPath)).map { changeset =>
           compile(module)
             .map { _ =>
+              // Side-effect: ensures that module's source is correctly initialized.
```

Comment:
What happens when this side-effect isn't performed?

Reply:
We get a failed execution `The Main.main source not found.`